Neos.ContentRepository/Classes/Domain/Model/Workspace.php

```diff
@@ -58,7 +58,7 @@ class Workspace
      * @ORM\Column(type="text", length=500, nullable=true)
      * @Flow\Validate(type="StringLength", options={ "minimum"=0, "maximum"=500 })
      */
-    protected $description;
+    protected $description = '';
```

Comment:
Remove the `nullable=true` and create a migration, to pull through. 😎 Or leave it as nullable in the code here and below…